consul/conf.yaml.example

```diff
@@ -54,6 +54,10 @@ instances:
       #   - gunicorn
       #   - redis
 
+      # If you'd rather not maintain a whitelist, you can increase the maximum
```

Comment:
Let me know if you'd like me to change the wording. I just wanna get the option in the example to save someone else the 3 hours it took me to figure out why our stats were off